independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/messagewriter/MessageIcons.java

```diff
@@ -1,14 +1,25 @@
 package io.quarkus.devtools.messagewriter;
 
-public final class MessageIcons {
+import io.smallrye.common.os.OS;
 
-    public static final String OK_ICON = "\u2705";
-    public static final String NOK_ICON = "\u274c";
-    public static final String NOOP_ICON = "\uD83D\uDC4D";
-    public static final String WARN_ICON = "\uD83D\uDD25";
-    public static final String ERROR_ICON = "\u2757";
+public enum MessageIcons {
 
-    private MessageIcons() {
+    OK_ICON("\u2705", "[SUCCESS]"),
+    NOK_ICON("\u274c", "[FAILURE]"),
+    NOOP_ICON("\uD83D\uDC4D", "[FAILURE]"),
```

Comment:
```suggestion
    NOOP_ICON("\uD83D\uDC4D", ""),
```

Reply:
NOOP means no operation but doesn't mean failure at all. I would put an empty string her.

Reply:
```
applying codestarts...
>> java
>> maven
>> quarkus
>> config-properties
>> dockerfiles
>> maven-wrapper
>> resteasy-example

-----------
[FAILURE] quarkus project has been successfully generated in:
--> target\extensions-test
```

:)

Reply:
BTW I don't think for this message we should use a NOOP, it should be a OK.